code/game/machinery/teleporter.dm

```diff
@@ -67,11 +66,17 @@
 	ui_interact(user)
 
 /obj/machinery/computer/teleporter/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 1)
-	if(..())
-		return
```

Comment:
Given that `ui_interact` doesn't normally return anything this looks like an alright thing to remove

especially since `ui_interact` doesn't return anything. This was probably some leftover from `attack_hand` or something